dapp/src/index.js

```diff
@@ -2,6 +2,7 @@ import 'babel-polyfill'
 import 'utils/validation/yup'
 
 import React from 'react'
+import { PersistGate } from 'redux-persist/integration/react'
```

Comment:
cool, what's this for?

Reply:
@leonprou  Help to keep the jwt token persistent.